docker/Dockerfile.ruby_host

```diff
@@ -0,0 +1,23 @@
+FROM ruby:2.6-alpine as build
+
+ENV BUNDLER_VERSION 2.1.4
+ENV LANG=C.UTF-8 \
+    BUNDLE_JOBS=6 \
```

Comment:
are these env keys used anywhere?

Reply:
They're parameters for bundle to manage the download jobs.

Reply:
I'm guessing they're auto used then. No probs, I guess I've never configured bundler like this before. I half expected them to be used like the bundler version is.

Reply:
That was an extra I added over Maddie's rubocop example - to absolutely ensure it's installing the version we are expecting.